package.json

```diff
@@ -13,8 +13,10 @@
   "devDependencies": {
     "babel-cli": "^6.24.1",
     "babel-eslint": "^7.2.3",
+    "babel-plugin-module-resolver": "^2.7.1",
```

Comment:
This should be removed (unused in webpack.config)

Reply:
it is for root path resolving